packages/opam-bundle/opam-bundle.0.4/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+authors: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+license: "GPL-3"
+tags: "org:ocamlpro"
+homepage: "https://github.com/AltGr/opam-bundle"
+bug-reports: "https://github.com/AltGr/opam-bundle/issues"
+depends: [
+  "ocaml"
+  "ocamlfind"
```

Comment:
Is ocamlfind just a biuld dependency (as ocaml)?

Reply:
Yes, technically the package just installs  a binary that doesn't manage OCaml sources or artifacts, so everything should be a build dependency.

Reply:
(but then there is always the old dilemma of "should it still get recompiled if e.g. there is a fix in `opam-client` for a bug that affects it"... I guess in that case we could file a new release to force recompilation)

Reply:
I think that we should keep `cmdliner` and `opam-client` out, as you actively use the libraries (and indeed want the package to be rebuilt in case of bugfixes), while ocaml and ocamlfind are only invoked externally during the build

Reply:
I've made the change. I'll let the tests re-run before merging